force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +301,13 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+        Book myBook = new Book('A Brief History of Time', 'Stephen Hawking');
 
         // Call the getBookDetails method on the book object to get the details of the book
+       String bookDetails =  myBook.getBookDetails();
 
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return bookDetails; // Replace null with the variable you used to store the result
```

Comment:
good job. If you  to be fancy you could do this on on one line. So folks may not find it as readable though. 

```
  public static String createAndGetBookDetails() {
        return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();
    }
    ```